src/ravl/ravl.c

```diff
@@ -0,0 +1,551 @@
+/*
+ *
+ * Copyright (C) 2018-2024 Intel Corporation
```

Comment:
why `2018-`?

Reply:
The code of RAVL tree comes from https://github.com/pmem/pmdk/blob/master/src/core/ravl.c